cloudca/resource_cloudca_volume_test.go

```diff
@@ -0,0 +1,102 @@
+package cloudca
+
+import (
+	"fmt"
+	"testing"
+
+	cca "github.com/cloud-ca/go-cloudca"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccVolumeCreate(t *testing.T) {
+	t.Parallel()
+
+	instanceID := "6f26111d-464d-4fc8-9c72-7a181a96c257"
+	volumeName := fmt.Sprintf("terraform-test-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckVolumeCreateDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVolumeCreate(environmentID, instanceID, volumeName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckVolumeCreateExists("cloudca_volume.foobar"),
+				),
+			},
+		},
+	})
+}
+
+func testAccVolumeCreate(environment, instance, name string) string {
+	return fmt.Sprintf(`
+resource "cloudca_volume" "foobar" {
+	environment_id = "%s"
+	name           = "%s"
+	disk_offering  = "fd78763c-f33a-43f3-b1e3-63bf59a48350"
```

Comment:
maybe replace this with %s and have the disk offering ID on line 17 next to the instanceId constant just to keep any hardcoded ID values together in case they need to be changed in the future